manifests/k/KDE/Dolphin/Nightly/master-b6ec4b1c9/KDE.Dolphin.Nightly.installer.yaml

```diff
@@ -2,15 +2,15 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: KDE.Dolphin.Nightly
-PackageVersion: master-b6ec4b1c9
+PackageVersion: master-271
 Scope: machine
 InstallModes:
 - interactive
 - silent
 Installers:
 - Architecture: x64
   InstallerType: nullsoft
-  InstallerUrl: https://binary-factory.kde.org/job/Dolphin_Nightly_win64/271/artifact/dolphin-master-271-windows-msvc2019_64-cl.exe
+  InstallerUrl: https://binary-factory.kde.org/view/Windows%2064-bit/job/Dolphin_Nightly_win64/lastSuccessfulBuild/artifact/dolphin-master-271-windows-msvc2019_64-cl.exe

```

Comment:
Are previous versions of nightly builds removed? If they are, then using the last successful build is good. If they are not, then we should keep using the specific build numbers to maintain the version history and keep previous versions available through winget

Reply:
They are for KDE projects. These are hard to maintain because the file name changes too.